### ISSUE TITLE
Cant Subscribe to Smart Home Devices Management

### ISSUE THREAD

**Issue by shezxx** [codetheweb/tuyapi#516]
![image](https://user-images.githubusercontent.com/89923202/133891748-18817732-a9d3-4708-856d-72d3c9c680b1.png)

Followed the Setup Instructions, and created a Project but cant seem to subscribe to any APIs.

Also, 
Tuya like to keep changing thing, when creating a project, PaaS is no longer an option. The two options available are custom and smart home

**Comment by helixd-2k18** [comment]
I had same problem, I registered new account...

**Comment by shezxx** [comment]
Yep, did try that. With the new account I've created I still cant subscribe to Smart Home Devices Management, interestingly though I could subscribe to **Industrial Devices Management** which I was really excited until I figured out it was useless

**Comment by peleccom** [comment]
I have The same issue. Can't do anything

**Comment by codetheweb** [comment]
Sounds like something on Tuya's end, let me know if I can help out at all.

**Comment by dnrajpara** [comment]
Facing same issue. I can't subscribe (again). Looks like my project has authorised the API but does not look like I can use it. When I tried using tuya-cli (version 1.17.2), I face error "There was an issue fetching that device. Make sure your account is linked and the ID is correct."

**Comment by peleccom** [comment]
I was able to subscribe few days later. 
I think it were temporary issues on tuya servers or account is restricted first two-three days after registration

**Comment by dnrajpara** [comment]
> I was able to subscribe few days later.
> I think it were temporary issues on tuya servers or account is restricted first two-three days after registration

Thanks for the tip. I did try again it worked..!! Yippee..!!

**Comment by evlo** [comment]
https://github.com/tuya/tuya-home-assistant/issues/815 possibly related?
